arm-software/embedded/scripts/build_atfe_leak_ubsan_sanitizer.sh

```diff
@@ -7,6 +7,9 @@
 
```

Comment:
I suggest you use the same file suffix to both "build_" and "test_", like:
`build_atfe_sanitizer.sh` and `test_atfe_sanitizer.sh`